en/docs/modules/misc/KeyboardChip.md

```diff
@@ -152,6 +164,37 @@ Input names can be accessed by `KeyboardChip.`nameOfInput, and are listed below.
 
 ## Remarks
 
+### How to use
```

Comment:
As there's the huge list of input names between these, maybe specify how to use what?
```suggestion
### How to use the event
```